src/utils/workBannerUtility.ts

```diff
@@ -0,0 +1,26 @@
+import { WorkBannerConsumerProps } from 'HOCs/WorkBannerProvider';
+
+export const ACCOUNT_VERIFICATION = 'Click here to verify your email and phone number to start booking!';
+
+export const showAccountVerificationBanner = (
+  isUserVerified: boolean,
+  bannerDispatch: WorkBannerConsumerProps['bannerDispatch'],
+  bannerState: WorkBannerConsumerProps['bannerState']): void => {
+  const userVerified = isUserVerified && bannerState.show;
```

Comment:
this can confusing: `userVerified` is equal to `isUserVerified` AND a banner state.
I don't understand why userVerified is conditional on the banner state which is a UI variable.

Did you mean `userVerified` be named something else?

Reply:
That's a good point.  We're receiving the return of `hasCompletedVerification(user)` which will tell us if a user has finished phone and email verification.  so if the user is verified and the banner is shower we want to close the banner, and the opposite if the user is unverified.  I think I can fix up the names a bit to make it more clear 🙏